packages/template-retail-react-app/app/components/store-locator/list.jsx

```diff
@@ -24,7 +24,8 @@ export const StoreLocatorList = () => {
     const [initialSelectedStoreId, setInitialSelectedStoreId] = useState(selectedStoreId)
 
     const hasItemsInBasket = derivedData?.totalItems > 0
-    const storeSelectionDisabled = MULTISHIP_IS_ENABLED ? false : hasItemsInBasket
+    const multishipEnabled = getConfig()?.app?.multishipEnabled
```

Comment:
```suggestion
    const multishipEnabled = getConfig()?.app?.multishipEnabled ?? true
```
If `getConfig()?.app?.multishipEnabled` is not defined, should we set it true here?

Reply:
@vmarta good suggestion. done!